transactions/epoch/admin/reset_epoch.cdc

```diff
@@ -1,24 +1,22 @@
 import FlowEpoch from 0xEPOCHADDRESS
 import FlowIDTableStaking from 0xIDENTITYTABLEADDRESS
 
-transaction(randomSource: String,
+transaction(currentEpochCounter: UInt64,
+            randomSource: String,
             newPayout: UFix64,
             startView: UInt64,
             endView: UInt64,
-            collectorClusters: [String]
-            clusterQCs: [String],
-            dkgPubKeys: [String]) {
+            collectorClusters: [String]) {
```

Comment:
If it is safe from the smart contract perspective to omit the `collectorClusters` input as well I think we should.